production.Dockerfile

```diff
@@ -9,6 +9,7 @@ SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
 RUN apt-get update \
     && apt-get install -y --no-install-recommends 'curl=7.*' 'git=1:2.*' 'build-essential=12.*' \
+    && apt-get install -y --no-install-recommends 'pkg-config=0.*' 'libxml2-dev=2.*' 'libxmlsec1-dev=1.*' \
```

Comment:
I could use a look from Deployments Team, I'm particularly not sure if we can get rid of `pkg-config` after installing the other deps. These requirements come from https://github.com/mehcode/python-xmlsec#install

POV? @fuziontech @macobo @tiina303

Reply:
We only need `pkg-config` when compiling packages, so it can be added to the `apt purge` line below.